packages/server-side-graphql-client/tests/index.test.js

```diff
@@ -0,0 +1,224 @@
+const { Text, Integer } = require('@keystonejs/fields');
+const { multiAdapterRunners, setupServer } = require('@keystonejs/test-utils');
+
+const {
+  createItems,
+  createItem,
+  deleteItem,
+  deleteItems,
+  getItem,
+  getItems,
+  updateItem,
+  updateItems,
+} = require('../lib/server-side-graphql-client');
+
+const testData = [
+  {
+    data: {
+      name: 'test',
+      age: 30,
+    },
+  },
+  { data: { name: 'test2', age: 40 } },
+];
+const schemaName = 'testing';
+
+const seedDb = ({ keystone }) =>
+  createItems({
+    keystone,
+    listName: 'Test',
+    items: testData,
+    schemaName,
+  });
+
+function setupKeystone(adapterName) {
+  return setupServer({
+    adapterName,
+    createLists: keystone => {
+      keystone.createList('Test', {
+        fields: {
+          name: { type: Text },
+          age: { type: Integer },
+        },
+      });
+    },
+  });
+}
+
+multiAdapterRunners().map(({ runner, adapterName }) =>
+  describe(`Adapter: ${adapterName}`, () => {
+    describe('create', () => {
+      test(
+        'Should create and get single item',
```

Comment:
```suggestion
        'createItem, getItem: Should create and get single item',
```

Let's include the name of the function under test in the test name. Very helpful when things go wrong.